terraform/s3.tf.json.template.py

```diff
@@ -19,6 +16,27 @@
                             "days": 1
                         }
                     }
+                },
+                "public_bucket": {
+                    "bucket": config.s3_public_bucket,
+                    "acl": "public-read",
+                    "website": {
+                        "index_document": "error.html"
```

Comment:
Does this file need to exist in the bucket? If so can we create it with Terraform?

https://www.terraform.io/docs/providers/aws/r/s3_bucket_object.html

Reply:
If it's intentional that the file doesn't exist, a comment should explain that.

Reply:
It's intentionally non-existent so that the URL will give a 404.  It's there just because it's a required field.  We can also create an object through terraform if we wanted to.